### PR TITLE
nxos_vpc updates

### PR DESCRIPTION
**Pull request by Qalthos**
##### SUMMARY
Clean up nxos_vpc

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
nxos_vpc

##### ANSIBLE VERSION

```
2.4
```

### CODE REVIEW

File: lib/ansible/modules/network/nxos/nxos_vpc.py [hunk 1]
```diff
@@ -228,7 +186,8 @@ def get_vpc(module):
         pkl_vrf = None
         peer_gw = False
 
-        run = get_vpc_running_config(module)[0]
+        run = execute_show_command('show running section vpc', module,
```
Comment (trishnaguha) [inline lib/ansible/modules/network/nxos/nxos_vpc.py]: Can we use `get_config` instead?

Comment (Qalthos) [reply lib/ansible/modules/network/nxos/nxos_vpc.py]: Good catch, let's find out!

**Comment by mikewiebe** [comment]
@Qalthos FYI... Saw this PR to refactor the module.  Decided to take it for a test drive to make sure the idempotence issue I fixed last week was ok but now the module errors out for both transports.

```
2017-06-12T15:48:58: %LIB-INFO: The full traceback is:
2017-06-12T15:48:58: %LIB-INFO: Traceback (most recent call last):
2017-06-12T15:48:58: %LIB-INFO:   File "/tmp/ansible_4uPW0u/ansible_module_nxos_vpc.py", line 371, in <module>
2017-06-12T15:48:58: %LIB-INFO:     main()
2017-06-12T15:48:58: %LIB-INFO:   File "/tmp/ansible_4uPW0u/ansible_module_nxos_vpc.py", line 336, in main
2017-06-12T15:48:58: %LIB-INFO:     if pkl_vrf.lower() not in get_vrf_list(module):
2017-06-12T15:48:58: %LIB-INFO:   File "/tmp/ansible_4uPW0u/ansible_module_nxos_vpc.py", line 147, in get_vrf_list
2017-06-12T15:48:58: %LIB-INFO:     vrf_table = body['TABLE_vrf']['ROW_vrf']
2017-06-12T15:48:58: %LIB-INFO: TypeError: string indices must be integers
2017-06-12T15:48:58: %LIB-INFO: fatal: [n9k-109.cisco.com]: FAILED! => {
2017-06-12T15:48:58: %LIB-INFO:     "changed": false, 
2017-06-12T15:48:58: %LIB-INFO:     "failed": true, 
2017-06-12T15:48:58: %LIB-INFO:     "module_stderr": "Traceback (most recent call last):\n  File \"/tmp/ansible_4uPW0u/ansible_module_nxos_vpc.py\", line 371, in <module>\n    main()\n  File \"/tmp/ansible_4uPW0u/ansible_module_nxos_vpc.py\", line 336, in main\n    if pkl_vrf.lower() not in get_vrf_list(module):\n  File \"/tmp/ansible_4uPW0u/ansible_module_nxos_vpc.py\", line 147, in get_vrf_list\n    vrf_table = body['TABLE_vrf']['ROW_vrf']\nTypeError: string indices must be integers\n", 
2017-06-12T15:48:58: %LIB-INFO:     "module_stdout": "", 
2017-06-12T15:48:58: %LIB-INFO:     "msg": "MODULE FAILURE", 
2017-06-12T15:48:58: %LIB-INFO:     "rc": 0
2017-06-12T15:48:58: %LIB-INFO: }

```

**Comment by Qalthos** [comment]
@mikewiebe thanks for the info, that shouldn't be too much trouble

Edit: Yup, I was too aggressive with run_commands, that's back out now.

**Review by trishnaguha — APPROVED** [review]

**Pull request merged**